index.html

```diff
@@ -54,6 +55,174 @@ <h2 class="connect">LET'S CONNECT</h2>
+				<div class="grid-container2">
+				  <div class="grid-twelve">
+				    <p>CANOPY</p>
+				</div>
+				  <div class="grid-twentytwo">
+					<p>.</p>
+				 </div>
+				  <div class="grid-thirtytwo">
+					<p>Back End Dev</p>
+				 </div>
+				  <div class="grid-fortytwo">
+					<p>.</p>
+				 </div>
+				  <div class="grid-fiftytwo">
+					<p>2015</p>
+				 </div>
+				</div>
+				<p class="day2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="langs2">
+					<li class="btn12">html</li>
+					<li class="btn22">css</li>
+					<li class="btn32">javaScript</li>
+				</ul>
+				<button class="btn42" type="submit">See Project</button>
+			</div>
+			<div class="works3">
+				<img class="snap3" src="images/SnapshootPortfolio.svg" alt="">
+				<h2 class="tonic3">Tonic</h2>
+				<div class="grid-container3">
+				  <div class="grid-13">
+					<p>CANOPY</p>
+				  </div>
+				  <div class="grid-23">
+					<p>.</p>
+				  </div>
+				  <div class="grid-33">
+					<p>Back End Dev</p>
+				  </div>
+				  <div class="grid-43">
+					<p>.</p>
+				  </div>
+				  <div class="grid-53">
+					<p>2015</p>
+				  </div>
+				</div>
+
+				<p class="day3">
+					A daily selection of privately personalized reads; 
+					no accounts or sign-ups required.
+				</p>
+				<ul class="langs3">
+					<li class="btn13">html</li>
+					<li class="btn23">css</li>
+					<li class="btn33">javaScript</li>
+				</ul>
+				<button class="btn43" type="submit">See Project</button>
+			</div>
+			<div class="works4">
+				<img class="snap4" src="images/snapshot4.svg" alt="">
+				<h2 class="tonic4">Multi-Post Stories</h2>
+				<div class="grid-container4">
+				  <div class="grid-14"><p>CANOPY</p></div>
+				  <div class="grid-24"><p>.</p></div>
+				  <div class="grid-34"><p>Back End Dev</p></div>
+				  <div class="grid-44"><p>.</p></div>
+				  <div class="grid-54"><p>2015</p></div>
+				</div>
+				<p class="day4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="langs4">
+					<li class="btn14">html</li>
+					<li class="btn24">css</li>
+					<li class="btn34">javaScript</li>
+				</ul>
+				<button class="btn44" type="submit">See Project</button>
+			</div>
+		</section>
+	<div class="div-class">
+		<section class="about-myself">
+      		<div class="desk-about">
+        		<h1 class="aboutmy">About <br> Myself</h1>
+        		<p class="am">I’m a software developer! I can help you build a product , feature or website Look through some of my
+	          	work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+	          	contact me.</p>
+        		<button type="button" class="connect">LET'S CONNECT</button>
+        		<ul class="icons">
+                    <li class="twitter"><img src="images/github.svg" alt="github svg"></li>
+					<li class="vector1"><img src="/images/vector.svg" alt="vector"></li>
```

Comment:
- [ ] Kindly fix this image as it seems it's not properly referenced.
![Capture](https://user-images.githubusercontent.com/85095702/185766144-f25fda18-3abd-449e-9b73-a60a2d689e65.PNG)

- [ ] Also, kindly remove the s circled above